RELEASE-NOTES.md

```diff
@@ -26,6 +26,17 @@
 * Fixes `snow connection add` behavior when `connections.toml` file exists.
 * Add more readable error messages in `snow object create` command.
 
+
+# v3.0.1
+## Backward incompatibility
```

Comment:
Let's put this under unreleased. 3.0.0 release (and rc) should be done before this change

Reply:
Moved under Unreleased version section.